test/conformance/gateway_conformance_test.go

```diff
@@ -52,46 +57,45 @@ func TestGatewayConformance(t *testing.T) {
 	var (
 		skippedTests      []string
 		supportedFeatures []suite.SupportedFeature
+		mode              string
 	)
 	switch rf := testenv.KongRouterFlavor(); rf {
 	case dpconf.RouterFlavorTraditionalCompatible:
 		skippedTests = skippedTestsForTraditionalRoutes
 		supportedFeatures = traditionalRoutesSupportedFeatures
+		mode = "traditional-compatible"
 	case dpconf.RouterFlavorExpressions:
 		supportedFeatures = expressionRoutesSupportedFeatures
+		mode = "expressions"
```

Comment:
There are `const`s for those 2

https://github.com/Kong/kubernetes-ingress-controller/blob/2cbf6313b96c6b0e9c3adc856070472ae1b68c6d/internal/dataplane/config/router.go#L5-L9